extension/llm/modules/test/test_mha.py

```diff
@@ -92,7 +93,11 @@ def test_attention_eager(self):
         et_res = self.et_mha(self.x, self.x)  # Self attention.
         tt_res = self.tt_mha(self.x, self.x)  # Self attention.
 
-        self.assertTrue(torch.allclose(et_res, tt_res))
+        assert_close(
+            et_res,
+            tt_res,
+            msg=f"TorchTune output is not close to ET output.\n\nTorchTune: {tt_res}\nET output: {et_res}",
```

Comment:
I don't think you need the messages, but not blocking land over them

Reply:
Would have been useful when I was debugging the error imo

Reply:
yes but I think assert_close gives you nice messages